src/Tests/Tests/CommonOptions/DateMath/DateMathExpressions.doc.cs

```diff
@@ -232,5 +232,21 @@ public class DateMathExpressions
 			(twelveMonths > fiftyTwoWeeks).Should().BeTrue();
 			(oneYear > fiftyTwoWeeks).Should().BeTrue();
 		}
+		[U]
+		public void DifferentLocales()
```

Comment:
Can use `UseCultureAttribute` on test method, and separate out the three different culture tests into separate unit tests. Since these are in a `*.doc.cs` file, these should also be marked with `// hide` to not be emitted in generated documentation e.g.

```c#
// hide
[U]
[UseCulture("sv-SE")]
public void SerializesWithInvariantCulture()
{
   // ..implementation
}
```

Reply:
Cool stuff! Didn't know about that!

Reply:
Now I see, the UseCultureAttribute is copied from xunit samples and available only in Tests.Reproduce. I will move the unit test there unless you think it's a bad idea @russcam

Reply:
ah, gotcha. I think moving it to `Tests.Core` project, perhaps under `Xunit` directory might make most sense? That way, both Tests.Reproduce and Tests can use it